src/Gothic/Externals.hpp

```diff
@@ -6,6 +6,11 @@ namespace GOTHIC_NAMESPACE
 {
     int BassMusic_Play()
     {
+        if (s_musicSystemDisabled)
```

Comment:
Compilation error
```
D:\a\zBassMusic\zBassMusic\src\Gothic/Externals.hpp(9): error C2065: 's_musicSystemDisabled': undeclared identifier
D:\a\zBassMusic\zBassMusic\src\Gothic/Externals.hpp(9): error C2065: 's_musicSystemDisabled': undeclared identifier
D:\a\zBassMusic\zBassMusic\src\Gothic/Externals.hpp(9): error C2065: 's_musicSystemDisabled': undeclared identifier
D:\a\zBassMusic\zBassMusic\src\Gothic/Externals.hpp(9): error C2065: 's_musicSystemDisabled': undeclared identifier
```

Reply:
@damianut, please make the following changes:

1. Replace `s_musicSystemDisabled` with `zCMusicSystem::s_musicSystemDisabled` - this symbol is a static member of zCMusicSystem class, but in external we are outside the class context.
2. Move your `if` below `parser->GetParameter(id);` - in externals, we must always pop every parameter from this stack before early return. Otherwise, it would lead to memory leaks or even a crash.

The final code should look like
```cpp
    int BassMusic_Play()
    {
        zSTRING id;
        parser->GetParameter(id);
        if (zCMusicSystem::s_musicSystemDisabled)
        {
            log->Error("Music system disabled.");
            return 0;
        }
        zCMusicTheme* theme = zmusic->LoadThemeByScript(id);
        zmusic->PlayTheme(theme, zMUS_THEME_VOL_DEFAULT, zMUS_TR_DEFAULT, zMUS_TRSUB_DEFAULT);
        return 0;
    }
```